tests/fixtures/test_rst_svg_attrs.html

```diff
@@ -0,0 +1 @@
+<img alt="alternate text" src="https://example.com/badge.svg">
```

Comment:
We're kind of experiencing a regression here, right? Before, this:

```rst
.. image:: https://example.com/badge.svg
   :height: 100px
   :width: 25.0%
   :alt: alternate text
   :align: right
```

would become:

```html
<img align="right" alt="alternate text" height="100px" src="https://example.com/badge.svg" width="25.0%">
```

but now it becomes:

```html
<img alt="alternate text" src="https://example.com/badge.svg">
```

Reply:
Yes, but the current behavior is problematic:
 - `align="right"` does not actually align the image,  [CSS](https://github.com/pypa/warehouse/pull/4391/files) is necessary for that.
 - `height="100px"` wont have any effect, because units (`px`) must be added to CSS, not image attributes
 - Those attributes are preserved for SVG images only (or more precisely for URLs ending ".svg")

https://github.com/pypa/readme_renderer/pull/114 addresses image alignment and dimensions. Without unifying SVG with other image types, the subsequent PR would fix styling of non-SVG images only.

(I created two PRs because the second relaxes what HTML is allowed, so there can be more discussion.)

Reply:
Got it, thanks for the explanation.